iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -158,4 +158,10 @@
     <PackageReference Condition=" '$(Configuration)' == 'Debug' " Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Why do we need AutoRest dependency on our existing service client?

Reply:
We don't need it, I'll remove it

Reply:
I've removed it now, good catch